e2e-tests/insurance/cypress/e2e/journeys/check-your-answers/policy/change-your-answers/change-your-answers-policy-type-change-single-to-multiple-policy-type-summary-list.spec.js

```diff
@@ -106,23 +106,23 @@ context('Insurance - Change your answers - Policy and exports - Change single to
       it(TOTAL_MONTHS_OF_COVER, () => {
         fieldId = TOTAL_MONTHS_OF_COVER;
 
-        const expectedValue = `${application.POLICY_AND_EXPORTS[fieldId]} months`;
+        const expectedValue = `${application.POLICY[fieldId]} months`;
```

Comment:
not a blocker.  What I have started doing in tests is dereferencing { POLICY } = application at the top to save some space